.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+<!--
```

Comment:
Should any of these be checklists instead of bullet points?

Reply:
I didn't see any of them as simple check-offs, since the instructions are asking for information. Perhaps, "[ ] I have written a docstring/readme/adr" could be a check mark, but I'd rather have a link to it.